webapp/src/components/systemConsole/enableSurvey/index.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useState} from 'react';
+
+import type {CustomSettingChildComponentProp} from 'components/systemConsole/index';
+
+import './style.scss';
+
+function EnableSurvey({id, setSaveNeeded, onChange, config, setInitialSetting}: CustomSettingChildComponentProp) {
+    const defaultValue = false;
```

Comment:
Yeah I'd agree this does seem unnecessary unless we're planning on changing it.

Reply:
I'm using it in two places right now so want to have the default value in a constant to serve as a single place of truth instead of the value being spread around.

Agree on moving it outside the component as uppercase snake case. Done!